src/components/ha-hls-player.ts

```diff
@@ -77,17 +70,9 @@ class HaHLSPlayer extends LitElement {
   protected updated(changedProps: PropertyValues) {
     super.updated(changedProps);
 
-    const attachedChanged = changedProps.has("_attached");
     const urlChanged = changedProps.has("url");
 
-    if (!urlChanged && !attachedChanged) {
-      return;
-    }
-
-    // If we are no longer attached, destroy polyfill
-    if (attachedChanged && !this._attached) {
-      // Tear down existing polyfill, if available
-      this._destroyPolyfill();
+    if (!urlChanged) {
```

Comment:
I think it is a bit overkill to destroy HLS when the URL changes btw, (not sure if it can even happen when we are attached?) we can just call `loadSource`?

Reply:
Yes, you are probably right, but I don't have time to test it right now (I don't really have a frontend dev setup - I'm just typing my edits in the github web interface 😄 ), and we need to consider the other players as well. I'm not sure when we get URL changes actually.
Maybe we can just fix the bug first and improve it later.

Reply:
BTW, `loadSource` wasn't sufficient and the destroying was necessary before v1.0, but that behavior is supposed to be fixed now. https://github.com/video-dev/hls.js/projects/6#card-58632654
However, I tried it (a simple `loadSource` call with a new url) and it didn't seem to work...